### ISSUE TITLE
[SPIKE] Investigate the "Feedback" Button to Determine if we want to Include it

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#63308]
## Tasks
- [ ] Find out why we took it out in the first place
- [ ] Investigate the "Feedback" Button to determine how we can get the input if we re-implement the button
- [ ] Share with the team

**Comment by benbrasso-agile6** [comment]
### TL;DR

* So, I believe we're investing this ticket because 1 or 2 Veterans said it might be nice to have something at the end to provide feedback. For our team, it comes down to is the time commitment analyzing Medallia data valuable to our team or does the time investment equate to the same value the data will bring to us. And, I think that answer is no. Therefore, I don't recommend we re-adding Medallia into the mix. 

### Findings from digging 

So, there was an initial request from Patrick to remove it from all pages: https://dsva.slack.com/archives/C022AC2STBM/p1629382828348700?thread_ts=1629382816.348500&cid=C022AC2STBM

I can def understand the desire to "remove it from all pages before launch," from Patrick. E.g., don't let it get in the way of finishing the form/process. 

It was removed by this PR/ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/28972

But, then, in summer 2022, Patrick or our team got interested in tracking product success. So, Patrick started some convo's with Jason Woodman from Contact Center. https://dsva.slack.com/archives/C022AC2STBM/p1656347286652589?thread_ts=1653592379.946939&cid=C022AC2STBM

Sometime before or after this exchange, the feedback button was reappearing on all product pages. 

And, as a result of that conversation with Jason Woodman, we gave Jason some keywords and/or pages to pull data from Medallia. Patrick saved 2 reports in onedrive, which I have access to. That data looks like this: 

![Screenshot 2023-08-11 at 2.15.31 PM.png](https://images.zenhubusercontent.com/62264cddfdd7546d65710eff/f1b434a3-5f87-4921-9bff-8987040596fa)

As far as I can tell, we no longer have the feedback button on our pages... again. 

So, this data is quite similar to the data we receive from VSignals. It's hard to tell, without further examination which is more valuable to our product. E.g. VSignals vs. Medallia. Plus, what's the volume of comments comparison. 

If we did want to request Medallia data, we'd start the process here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jwoodman5&labels=VSP-contact-center&template=qualitative-data-request.yml&title=Qualitative+Data+Request

So, I don't know what happen next. Patrick got really into looking at overall utilization and pulling reports directly from CDW. That is, utilization became more important to look at. 

But, he was still pulling VSignals reports for us and sharing: https://dsva.slack.com/archives/C022AC2STBM/p1658254053648239?thread_ts=1652386398.232339&cid=C022AC2STBM

Then, Patrick encouraged us to take over the VSignals analysis and that's where we are today. 

### Recommendations: 

* I don't think it's valuable for us to re-spend time evaluating VSignals vs Medallia
* I don't think adding analysis is valuable to our product and resources at this time, especially since the data from both systems is very similar 
* Therefore, I don't recommend we re-add the feedback button, even if it was to the confirmation page.

**Comment by benbrasso-agile6** [comment]
Moving into validate for @loripusey and @cgednalske to review

**Comment by loripusey** [comment]
I agree with your assessment. However, if we do ever add it back, I would suggest dropping VSignal analysis in favor of Medallia.

**Comment by cgednalske** [comment]
Agree with you both. 👍 And we could always keep the topic in mind in case any relevant questions fit in future research.

**Comment by benbrasso-agile6** [comment]
@kristenmcconnell @klawyer @stephenBarrs our team did a quick spike this week on the possibility of re-enabling the Medallia "Feedback" button on PCI (likely, on the last page of pre-check-in or check-in). @kristenmcconnell mentioned that this information might be helpful. You can view our analysis in the comments above. From our team, this spike is closed as of now.

**Comment by loripusey** [comment]
@kristenmcconnell @klawyer One way to keep track of cross-team features would be to create a GitHub project for OCTO, similar to how we did for the Mobile-CIE mobile check-in feature
